zomes/rea_agent/lib/src/lib.rs

```diff
@@ -27,9 +29,29 @@ pub fn handle_create_agent<S>(entry_def_id: S, agent: CreateRequest) -> RecordAP
     where S: AsRef<str>
 {
     let (header_addr, base_address, entry_resp): (_,_, EntryData) = create_record(&entry_def_id, agent)?;
+    // TODO: check if contains `link_pub_key` field
```

Comment:
This one will be contingent on our decision on how to approach who/how/when the key and entry get linked

Reply:
So instead of affiliating this here, create a new zome fn